### PR TITLE
B 19681 int

### PR DESCRIPTION
**Pull request by CoryKleinjanCACI**
## Summary

This task updates the 'About' screen when a Customer uploads ppm documents from zip code to full address. An issue was noticed during development due to the 'actualZipCode' being stored separately from the zip code originally entered by the user. The database currently does not have support for an actualAddress to be stored. After discussion with the customer it was decided that the actual zip code will still be saved separately, but the address field will just overwrite whatever the customer originally entered. This will be updated to support actualAddress in follow-on tasks. 

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Customer application
2. Login as a user 
3. Create a new move with a PPM and submit
4. Log out
5. Log in to the Office application
6. Login as a service counselor
7. Locate the move from step 3
8. Submit move details
9. Log out
10. Log in to the Customer application
11. Log in as the same user from step 2
12. Select the move from step 3
13. Click 'Upload PPM Documents'
14. Fill in the fields to include pickup and destination address (Use something different to validate that the update works)
15. Submit the form and complete the ppm
16. Log out
17. Access the Office Application
18. Login as a service counselor
19. Locate the move from step 3
20. View the PPM and verify that the address fields were updated.

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

**Comment by MInthavongsay** [comment]
Office details for move doesnt seem to be displaying full address. It's missing "address 3". Everything else looks  good.
Screenshot 2024-05-17 at 4 57 56 PM

**Comment by MInthavongsay** [comment]
Office details view not displaying full address. It's not displaying 'address 3".

Screenshot 2024-05-17 at 4 57 56 PM

File: src/components/Customer/PPM/Closeout/AboutForm/AboutForm.jsx [hunk 1]
```diff
@@ -43,25 +40,46 @@ const AboutForm = ({ mtoShipment, onBack, onSubmit, postalCodeValidator }) => {
...
 
   const initialValues = {
     actualMoveDate: actualMoveDate || '',
-    actualPickupPostalCode: actualPickupPostalCode || pickupPostalCode || '',
-    actualDestinationPostalCode: actualDestinationPostalCode || destinationPostalCode || '',
+    actualPickupPostalCode: '',
+    actualDestinationPostalCode: '',
+    pickupAddress: {
+      streetAddress1: '',
+      streetAddress2: '',
```
Comment (MInthavongsay) [inline src/components/Customer/PPM/Closeout/AboutForm/AboutForm.jsx:55]: Where are the streetAddress3s for all of these address objects? The Office details view is not correctly displaying the full address. Missing address3.

**Review by MInthavongsay — CHANGES_REQUESTED** [review]
See comments

File: src/components/Customer/PPM/Closeout/AboutForm/AboutForm.test.jsx [hunk 2]
```diff
@@ -12,20 +11,51 @@ beforeEach(() => {
...
-      actualMoveDate: '2022-05-19',
-      actualPickupPostalCode: '10001',
-      actualDestinationPostalCode: '60652',
-      hasReceivedAdvance: true,
-      advanceAmountReceived: 123456,
+      actualMoveDate: '31 May 2022',
+      actualPickupPostalCode: '',
+      actualDestinationPostalCode: '',
+      pickupAddress: {
+        streetAddress1: '812 S 129th St',
```
Comment (MInthavongsay) [inline src/components/Customer/PPM/Closeout/AboutForm/AboutForm.test.jsx:28]: these are all missing address3

File: src/components/Customer/PPM/Closeout/AboutForm/AboutForm.jsx (hunk 1 above)
Comment (CoryKleinjanCACI) [reply src/components/Customer/PPM/Closeout/AboutForm/AboutForm.jsx:55]: Address three not showing on the office page, for service counselors, appears to either be intentional or a bug in the application. shipmentDisplay.jsx is only set to show streetAddress1 and streetAddress2.

**Review by ajlusk — APPROVED** [review]
Looks good. Tests pass. Functions as desired.

I concur with Mike's request to add the address line 3. I worked another ticket where there was an opportunity (but not requirement) to start displaying line 3. Our former PO was also in favor of adding it at the time.

There was some odd behavior when testing this with more than one PPM but that appears to be unrelated to these changes.

**Comment by r-mettler** [comment]
Original addresses with address 3 field shown

image
image

Updated address looks good.
image

**Review by r-mettler — APPROVED** [review]

**Pull request merged**